runtime/context.go

```diff
@@ -64,18 +64,19 @@ const (
 )
 
 const (
-	scopeTagClientMethod    = "clientmethod"
-	scopeTagEndpointMethod  = "endpointmethod"
-	scopeTagClient          = "clientid"
-	scopeTagEndpoint        = "endpointid"
-	scopeTagHandler         = "handlerid"
-	scopeTagError           = "error"
-	scopeTagStatus          = "status"
-	scopeTagProtocol        = "protocol"
-	scopeTagHTTP            = "HTTP"
-	scopeTagTChannel        = "TChannel"
-	scopeTagsTargetService  = "targetservice"
-	scopeTagsTargetEndpoint = "targetendpoint"
+	scopeTagClientMethod         = "clientmethod"
+	scopeTagClientTargetEndpoint = "clienttargetendpoint"
```

Comment:
It is probably not necessary to introduce this new tag, `scopeTagsTargetEndpoint` ("targetendpoint") should be sufficient because of the existing contexts for the client operation. Also tchannel client and http client sharing this same tas make it easier when we query by tags.